include/aws/lambda-runtime/runtime.h

```diff
@@ -163,8 +163,6 @@ class runtime {
         std::string const& url,
         std::string const& request_id,
         invocation_response const& handler_response);
-
-private:
```

Comment:
What's the reason for making this public?

Reply:
clang-tidy pointed out that these were already made private a few lines up

ref: https://github.com/bmoffatt/aws-lambda-cpp/runs/4305830378?check_suite_focus=true
```
/home/runner/work/aws-lambda-cpp/aws-lambda-cpp/include/aws/lambda-runtime/runtime.h:167:1: error: redundant access specifier has the same accessibility as the previous access specifier [readability-redundant-access-specifiers,-warnings-as-errors]
private:
^~~~~~~~
/home/runner/work/aws-lambda-cpp/aws-lambda-cpp/include/aws/lambda-runtime/runtime.h:159:1: note: previously declared here
private:
^
```

Reply:
Ah! That's what I get for looking at this on a 5" phone screen.